app/services/record_declarations/retained.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module RecordDeclarations
+  module Retained
+    extend ActiveSupport::Concern
+
+    included do
+      extend RetainedClassMethods
+    end
+
+    module RetainedClassMethods
+      def required_params
+        %i[user_id cpd_lead_provider declaration_type declaration_date course_identifier evidence_held raw_event]
+      end
```

Comment:
Those are very similar to `StartedCompletedClassMethods` ones. For my understanding, do we not expect `evidence_held` to be present on non-retained declarations?

Reply:
Yes. This is currently the biggest difference between them.